romi-rover/control_panel/test/src/test_statemachine.cpp

```diff
@@ -1,6 +1,7 @@
 #include "gtest/gtest.h"
 #include "gmock/gmock.h"
 #include "StateMachine.h"
+#include "../mock/mock_statetransition.h"
```

Comment:
We might need to take a peak at the CMakeLists.txt there's not usually a need for a path for header files with CMake.